pages/+layout.svelte

```diff
@@ -0,0 +1,7 @@
+<script>
+  import { EvidenceDefaultLayout } from '@evidence-dev/core-components';
```

Comment:
You need to add the css and data too as well as the new prop

<script>
	import '@evidence-dev/tailwind/fonts.css';
	import '../app.css';
	import { EvidenceDefaultLayout } from '@evidence-dev/core-components';
	export let data;
</script>

<EvidenceDefaultLayout {data}>
	<slot slot="content" />
</EvidenceDefaultLayout>